Helper/Order.php

```diff
@@ -1095,7 +1052,16 @@ private function getProcessedRefunds($payment)
      */
     public function getTransactionState($transaction, $payment, $hookType = null)
     {
-        $transactionState = $transaction->type.":".$transaction->status;
+        $transactionType = $transaction->type;
+        // // If it is an apm type, it needs to behave as regular payment/credit.
+        // // Since there are previous states saved, it needs to mimic "cc_payment"/"cc_credit"
```

Comment:
nit: can remove the double comments? `// //`

Reply:
Upvote for nit 👍